src/trame_vtk/widgets/vtk/common.py

```diff
@@ -1130,3 +1131,60 @@ def set_camera(self, camera):
     @property
     def ref_name(self):
         return self._ref
+
+
+class VtkWebXRHelper(HtmlElement):
+    """
+        The VtkWebXRHelper component provides an easy way to add WebXR support to
+        a VtkView or a VtkLocalView. Just add this widget in a VtkView or
+        VtkLocalView widget and start an XR session with `startXR()`.
+        This widget will fire the `enterXR` and `exitXR` events when an XR session
+        is effectively entered and exited.
+
+    >>> webxr_helper = vtk.VtkWebXRHelper(
+    ...     ref=...,  # Identifier for this component
+    ...     draw_controllers_ray=True,  # Enable XR controllers rays
+    ... )
+    """
+
+    _next_id = 0
+
+    class XrSessionTypes(IntEnum):
+        """
+        WebXR Session Types supported by vtk.js
+        """
+
+        HmdVR = 0
+        MobileAR = 1
+        LookingGlassVR = 2
+        HmdAR = 3
+
+    def __init__(self, ref=None, **kwargs):
+        super().__init__(
+            "vtk-webXR-helper",
+            **kwargs,
+        )
+
+        if ref is None:
+            VtkWebXRHelper._next_id += 1
+            ref = f"trame__webxr_helper_{VtkWebXRHelper._next_id}"
+
+        self.__ref = ref
+        self._attributes["ref"] = f'ref="{ref}"'
+        self._attr_names += [("draw_controllers_ray", "drawControllersRay")]
+        self._event_names += [
+            "enterXR",
+            "exitXR",
+        ]
+
+    def startXR(self, session_type: XrSessionTypes) -> bool:
```

Comment:
this method should return a boolean.
Document what does the returned bool correspond to.